File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.js
function solution(numbers) {
    var answer = '';
    numbers_ = numbers.map(String);
    numbers_.sort((a, b) => (b+a) - (a+b))
    
    if (numbers_[0] === '0') {
        return '0';
    }
    
    answer = numbers_.join('');
    
    return answer;
}

File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.py
def solution(numbers):
    answer = ''
    numbers_str = list(map(str, numbers))
    numbers_str.sort(key=lambda x: x * 10, reverse=True)

    if numbers_str[0] == '0':
        answer = '0';
    else:
        answer = ''.join(numbers_str)
    return answer